.travis.yml

```diff
@@ -34,6 +43,13 @@ before_script:
   # Install linters
   - make install-linters
   - git submodule update --remote
+  # Install gcc 6 version, required for libskycoin
+  #FIXME $TRAVIS_TAG != ''
+  - if [[ $TRAVIS_TAG == 'osx' ]]; then
+    make tiny-firmware/vendor/libskycoin/Makefile;
```

Comment:
why not `make -C` ... ?

Reply:
@olemis this solve the project, then you can make use of the [`install-travis-gcc.sh`](https://github.com/fibercrypto/skywallet-mcu/pull/359/files#diff-354f30a63fb0907d4ad57269548329e3R50)

Reply:
make a phony target instead of that